### PR TITLE
Collected small changes and fixes

### PR DESCRIPTION
**Pull request by akohlmey**
**Summary**

This pull request combines multiple small changes and bug fixes.

**Related Issue(s)**

N/A

**Author(s)**

Axel Kohlmeyer, Temple University

**Licensing**

By submitting this pull request, I agree, that my contribution will be included in LAMMPS and redistributed under either the GNU General Public License version 2 (GPL v2) or the GNU Lesser General Public License version 2.1 (LGPL v2.1).

**Backward Compatibility**

N/A

**Implementation Notes**

The following individual changes are included:
- remove unused variables in fix qtpie/reaxff
- ensure that `Dump::sortcol` is initialized
- Clarify that Steve is the "inventor" of LAMMPS
- add note that SPH package detailed PDF doc is outdated
- updates for building flatpak bundles of LAMMPS-GUI
- bugfix for setting extvector for fixes in fix vector

**Post Submission Checklist**

- [ ] The feature or features in this pull request is complete
- [x] Licensing information is complete
- [x] Corresponding author information is complete
- [x] The source code follows the LAMMPS formatting guidelines
- [x] Suitable new documentation files and/or updates to the existing docs are included
- [x] The added/updated documentation is integrated and tested with the documentation build system
- [x] The feature has been verified to work with the conventional build system
- [x] The feature has been verified to work with the CMake based build system

### CODE REVIEW

File: doc/src/Intro_authors.rst [hunk 1]
```diff
@@ -27,7 +29,7 @@ lammps.org".  General questions about LAMMPS should be posted in the
    * - `Steve Plimpton <sjp_>`_
      - SNL (retired)
      - sjplimp at gmail.com
-     - MD kernels, parallel algorithms & scalability, code structure and design
+     - Original author, MD kernels, parallel algorithms & scalability, code structure and design
```
Comment (sjplimp) [inline doc/src/Intro_authors.rst]: original (all the other words under Areas of expertise are lower-case_

File: src/dump.cpp [hunk 2]
```diff
@@ -94,6 +94,7 @@ Dump::Dump(LAMMPS *lmp, int /*narg*/, char **arg) :
 
   maxbuf = maxids = maxsort = maxproc = 0;
   maxsbuf = 0;
+  sortcol = 0;
```
Comment (sjplimp) [inline src/dump.cpp]: Can you move this line up a dozen lines to be right after
sort_flag = 0

**Comment by sjplimp** [comment]
@akohlmey The intended logic with sorting was that if sort_flag was not set, sortcol was never accessed. Conversely, if sort_flag was set then sortcol was also set.  Was this violated somehow?

**Comment by akohlmey** [comment]
> @akohlmey The intended logic with sorting was that if sort_flag was not set, sortcol was never accessed. Conversely, if sort_flag was set then sortcol was also set. Was this violated somehow?

I cannot tell exactly how, but valgrind was complaining about unitialized data access which I could narrow down to sortcol being access uninitialized.

File: doc/src/Intro_authors.rst (hunk 1 above)
Comment (akohlmey) [reply doc/src/Intro_authors.rst]: done

File: src/dump.cpp (hunk 2 above)
Comment (akohlmey) [reply src/dump.cpp]: done

**Review by sjplimp — APPROVED** [review]

**Pull request merged**